recipes/scicookie/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "scicookie" %}
+{% set version = "0.6.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/scicookie-{{ version }}.tar.gz
+  sha256: e17a7641a8c0ed6bc27e7826a0db45db4d6aa9dfc5e939ca7718df6ffb5030ac
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv 
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6,<4
+    - poetry-core >=1.0.0
+    - pip
+  run:
+    - python >=3.6,<4
+    - cookieninja >=1.0.0
+    - sh >=2.0.4,<3.0.0  # [not win]
```

Comment:
@xmnlab is it optional? Doesn't look like based on the package metadata.

Reply:
actually this is not optional .. and probably we should skip the packaging for windows for now .. 
maybe we can think in a different approach in a near future in order to support windows as well.
thanks for catching that.

Reply:
In that case we cab remove the `# [not win]` and the `__win      # [win]` below.